generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/ComponentDestinationType.java

```diff
@@ -40,6 +42,11 @@ public ComponentDestinationType() {
      *                                  Component}
      */
     public Class<? extends Destination> inputTypeFor(final String component) {
+
+        if (components.containsKey(component)) {
```

Comment:
Had to move this up as IMHO, this is the correct location.
When it was down, it never get hit during tests as both `COMMAND_API` and `CUSTOM_COMMAND_API` will get fulfilled by the condition 
```
  if (component.contains(COMMAND_API)) {
            return components.get(COMMAND_API);
        }
```